casingSimulations/utils.py

```diff
@@ -3,25 +3,23 @@
 import time
 import datetime
 from discretize import utils
+import properties
 import casingSimulations
 
 
-def load_properties(filename, targetModule=None, targetClass=None):
+def load_properties(filename):
     """
     Open a json file and load the properties into the target class
+
+    As long as there are no namespace conflicts, the target __class__
+    will be stored on the properties.HasProperties registry and may be
+    fetched from there.
+
     :param str filename: name of file to read in
-    :param str targetClass: name of the target class to recreate
     """
     with open(filename, 'r') as outfile:
         jsondict = json.load(outfile)
-        module = (
-            targetModule if targetModule is not None else casingSimulations
-        )
-        if targetClass is None:
-            targetClass = getattr(
-                module, jsondict['__class__']
-            )
-        data = targetClass.deserialize(jsondict)
+        data = properties.HasProperties.deserialize(jsondict, trusted=True)
```

Comment:
`properties.HasProperties` has a `_REGISTRY` of all the HasProperties subclasses that have been imported (as long as there are no name conflicts with these classes...). If you specify `trusted=True`, `deserialize` will grab the correct `__class__` as long as it's been imported. This eliminates the need to specify module/class.

This assumes the class is imported and has a unique name - if this will not be the case, it may be worth reintroducing some of the module/class logic I deleted.